From a113bfbe35d894c92edde238c114a65c3400c00d Mon Sep 17 00:00:00 2001
From: Thomas Haller <thaller@redhat.com>
Date: Fri, 30 May 2014 12:39:00 +0200
Subject: [PATCH 1/1] fix make check failure due to error in translation config

Fix the following `make check` failure:

    make[1]: Entering directory `./NetworkManager/NetworkManager-0.8.1/po'
    rm -f missing notexist
    srcdir=. /usr/bin/intltool-update -m
    The following files contain translations and are currently not in use. Please
    consider adding these to the POTFILES.in file, located in the po/ directory.

    network-manager-applet-0.8.1/nm-applet.desktop.in
    ...
    network-manager-applet-0.8.1/src/wireless-security/wireless-security.c

    If some of these files are left out on purpose then please add them to
    POTFILES.skip instead of POTFILES.in. A file 'missing' containing this list
    of left out files has been written in the current directory.
    Please report to dcbw@redhat.com
    if [ -r missing -o -r notexist ]; then \
              exit 1; \
            fi
    make[1]: *** [check] Error 1

Signed-off-by: Thomas Haller <thaller@redhat.com>
---
 po/POTFILES.skip |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/po/POTFILES.skip b/po/POTFILES.skip
index e57048b..5614a81 100644
--- a/po/POTFILES.skip
+++ b/po/POTFILES.skip
@@ -3,3 +3,4 @@ examples/python/systray/eggtrayicon.c
 vpn-daemons/openvpn
 vpn-daemons/pptp
 vpn-daemons/vpnc
+network-manager-applet-0.8.1/
-- 
1.7.1

